src/core/evaluator.js

```diff
@@ -170,23 +170,22 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
            cs.isDefaultDecode(dict.getArray('Decode', 'D'));
   };
 
-  function PartialEvaluator(pdfManager, xref, handler, pageIndex,
-                            idFactory, fontCache, builtInCMapCache, options) {
-    this.pdfManager = pdfManager;
-    this.xref = xref;
-    this.handler = handler;
-    this.pageIndex = pageIndex;
-    this.idFactory = idFactory;
-    this.fontCache = fontCache;
-    this.builtInCMapCache = builtInCMapCache;
-    this.options = options || DefaultPartialEvaluatorOptions;
+  function PartialEvaluator(params) {
+    this.pdfManager = params.pdfManager;
```

Comment:
similar to import, can we make it via parameter deconstruction?

```
function PartialEvaluator({pdfManager, xref, handler, pageIndex, idFactory, fontCache,
                                            builtInCMapCache, options}) {
```

it will be nice to names of field accepted at the signature vs scrolling below to find out?